packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html

```diff
@@ -72,7 +74,53 @@ <h4 class="pt-3">
     data-expanded="true"
     data-heading="{{ 'telephony_line_softphone_configure_title' | translate }}"
 >
-    <p data-translate="telephony_line_softphone_configure_description"></p>
+    <div class="row mb-2">
+        <div class="col-md-12 mb-2">
+            <p
+                data-translate="telephony_line_softphone_configure_description"
+            ></p>
+            <oui-guide-menu data-text="{{ ::$ctrl.guide.title }}">
+                <oui-guide-menu-item
+                    data-href="{{ ::$ctrl.guide.url }}"
+                    data-external="$ctrl.guide.external"
+                    ><span data-ng-bind="::$ctrl.guide.name"></span>
+                </oui-guide-menu-item>
+            </oui-guide-menu>
+        </div>
+        <div
+            class="d-flex flex-wrap w-100 minw-0 align-items-center justify-content-start"
+        >
+            <oui-tile
+                data-ng-repeat="deviceId in ::$ctrl.softphoneDevices track by $index"
+                class="m-2"
+                heading="{{ 'Paul Iphone' }}"
```

Comment:
**nitpick:** Dont we need translations for this?

Reply:
Same as the other one